packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -164,8 +191,11 @@ const defaultL1Network: L1Network = {
   isArbitrum: false
 }
 
-const defaultL2Network: L2Network = {
+const defaultL2Network: ParentChain = {
   chainID: 412346,
+  partnerChainIDs: [
```

Comment:
ok now i'm confused by the same

`orbitChainIDs`?
`childChainIDs`?

speaking of hierarchy, they are not partners if they are different layers

but i think this is to follow the convention in nitro?
maybe leave a comment somewhere in the codebase as `parent` and `partner` could easily be mistaken as a typo

Reply:
It has to be kept like this to maintain the backwards compatibility in sdk